_posts/2017-08-21-spatial-data-10-4.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "WDC Spatial Data in Tableau 10.2"
+abstract: "Take advantage of Tableau's rich spatial features by connecting directly to GeoJSON data via the web data connector."
+---
+
+With the release of Tableau 10.4, you can now connect directly to spatial data.  Using the new geometry value in the [tableau.dataType]({{ site.baseurl }}/docs/api_ref#webdataconnectorapi.datatypeenum.float) enum, you can send GeoJSON objects directly back to Tableau.  
+
+This feature supports the following geoJson types:
+
+* Points and MultiPoints 
+* Lines and MultiLines
+* Polygons and MultiPolygons
+
+
+Learn more about how to upgrade to Tableau 10.4 and about the new features available here: [Get Tableau 10.4](https://www.tableau.com/new-features/10.4).
+
+Learn more about GeoJSON here: [GeoJSON documentation](http://geojson.org/).
+
+You can use the following hosted API scripts to take advantage of this new feature: 
+
+```
+https://connectors.tableau.com/libs/tableauwdc-2.3.latest.js
+https://connectors.tableau.com/libs/tableauwdc-2.3.latest.min.js
+```
```

Comment:
We need to update some of the GitHub pages  to list the 2.3 version https://github.com/tableau/webdataconnector/blob/master/docs/wdc_library_versions.md

In the compatibility table we need to add WDC 2.3.  I assume that 2.3 is for 10.4 (and later)